tests/unit/distribution/test_distribution.py

```diff
@@ -358,10 +358,8 @@ def test_external_provider_from_module_building(self, mock_providers):
                 description="test",
                 providers={
                     "inference": [
-                        Provider(
-                            provider_id="external_test",
+                        BuildProvider(
```

Comment:
how did this not get caught by https://github.com/meta-llama/llama-stack/pull/2917 ?

regardless, thank you for the fix.

Reply:
The unit-test workflow was failing silently on all PRs. This fix will now ensure unit tests will actually fail and block merge until the respective tests are updated or fixed.

Reply:
Here the unit-test failed for the mentioned PR #2917, but fails silently: https://github.com/meta-llama/llama-stack/actions/runs/16529125690/job/46749876207#step:4:183